sharding/common_submission_generator.jl

```diff
@@ -53,7 +53,8 @@ function generate_and_submit(submit_job_writer, cfg::JobConfig; input_file_name:
         end
         cp(joinpath("..", filename), joinpath(out_path, filename))
     end
-    cp(run_file, joinpath(out_path, basename(run_file)))
+    run_file_out = joinpath(out_path, basename(run_file))
+    cp(run_file, run_file_out)
```

Comment:
```suggestion
    run_file = joinpath(out_path, basename(input_file))
    cp(input_file, run_file)
```

trivial change (needs other changes in the script) but might clarify. The user provides an input file, we copy that somewhere else, and the copied file is the "run file".